prisma/util/lineshapes.py

```diff
@@ -0,0 +1,67 @@
+# © Copyright 2021, PRISMA’s Authors
+
+"""Function generators according to a linehsape.
+Each function generator constructs lambda functions by evaluating strings.
+The generator returns a function of npeaks number of profiles to be used for
+curve fitting
+The returned function is: returned_function(x, y0, h1, p1, w1, h2, p2, w2,
+h3, p3, w3...)
+"""
+
+
+def lorentzians(npeaks=1):
+    """Returns a function of npeaks number of lorentzian profiles"""
+
+    height_labels = ["h{}".format(n) for n in range(1, npeaks + 1)]
```

Comment:
Eventually it would be good to make the strings more readable using the `f"{}"` notation instead of `"{}".format()`. Example: `[f"h{n}" for n in range(1, npeaks + 1)]`

The same for the equation and expression strings.